dev-utils/poly-cli/index.js

```diff
@@ -125,7 +124,8 @@ yargs(hideBin(process.argv))
                 describe: "→ the type of feature: empty, preview, or importer",
             });
 
-            yargs.option("feature-version", {
+            // version is a reserved keyword in yargs
+            yargs.option("version", {
```

Comment:
I think this is not going to work, and is going to fail silently, precisely for that reason. I bumped into that when writing that.

Reply:
I created [this issue](https://github.com/yargs/yargs/issues/2199) there.

Reply:
It works. The trick is at the end of the command. Look for `.version(false)`.